web/package.json

```diff
@@ -8,51 +8,54 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
-    "@apidevtools/json-schema-ref-parser": "^9.0.1",
-    "@girder/components": "^2.2.5",
-    "@girder/oauth-client": "^0.7.6",
-    "@koumoul/vjsf": "1.21.0",
-    "@sentry/browser": "^5.20.1",
-    "@sentry/integrations": "^5.20.1",
-    "@vue/composition-api": "^1.0.0-beta.20",
-    "ajv": "^6.12.3",
-    "axios": "^0.19.2",
-    "core-js": "^3.6.5",
-    "filesize": "^6.1.0",
-    "js-yaml": "^3.14.0",
-    "lodash": "^4.17.19",
-    "moment": "^2.27.0",
+    "@apidevtools/json-schema-ref-parser": "^9.0.7",
+    "@girder/components": "~2.2.5",
```

Comment:
`@girder/components@2.3` has breaking changes to the import structure, so pin this minor version too.